dcos-labs/ui-kit
# feat: update React version

To facilitate `npm link` usage we will need to update the version of React for ui-kit.
This can be aligned with the version in Kommander, which is currently v17.0.2.

BREAKING CHANGE: a new major version of React has been installed

Closes D2IQ-81484



## Testing

1. Remove `node_modules`
2. Run `npm i` and confirm that the installs complete without issue
3. Run `npm start` and confirm that the app runs as expected

## Trade-offs



## Dependencies



## Screenshots



## Checklist for PR creator

- [ ] If any new components were added, there are exported from `packages/index.ts`
- [x] If this PR is associated with a JIRA, it is mentioned in commit message footer ("Closes …")
- [x] If this PR contains breaking changes, is stated in commit message body ("BREAKING CHANGE: …")
- [x] Info for applicable sections above is provided
